recipes/progress_reporter/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "progress_reporter" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name|lower }} 
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/marscher/progress_reporter
+  git_rev: {{ 'v' + version }}
+
+build:
+  noarch_python: True
+  number: 0
+# since this is noarch, only build on linux
+  skip: True  # [not linux]
+  script: python setup.py install
```

Comment:
It makes sure `setuptools` installs a flat egg (read ordinary directory as opposed to a compressed file) for use with another package manager. This commonly used building packages for `yum`, `apt-get`, etc.

If there are dependencies, it will also tell `setuptools` not to install them.